### DISCUSSION TITLE
Testing WoaS packages

### DISCUSSION THREAD

**Discussion by greythane** [WhiteCoreSim/WhiteCore-Dev#337] [Announcements]
As there has been an increase of interest expressed in WhiteCore, this release is to provide some **WhiteCoreSim-on-a-stick (WoaS)** packages to enable community testing and feedback   

The attached files are compiled and pre-configured instances of the standalone setup of WhiteCore for Windows, Mac OSX (Intel) and Linux (x64) and are based on the current 0.9.6-RC1 tagged commit    

All packages require the appropriate .NET or Mono to be installed  and have been pre-tested  
 Windows will most likely have the .NET libraries preinstalled  (4.8)
 OSX and Linux may need Mono installed (packages compiled with V6 on Mac and Linux)  
 
The packages also have some additional add-on modules that add functionality  
- Concierge for greeting newly arrived logins
- FlexibleWind for variable weather
- FractualTerrain for generating more complex terrain than the standard
- IRCChat for connecting to your local IRC server (or external chat server) for inworld <> external chats
- TidalWave to enable tide action on your island(s)
- TreePopulator for auto generating growing (and dying) trees

All packages are pre-configured as the 'WhiteCore-on-a-stick' variation with a user login and two regions available   
_Note:  The configuration of these packages can be re-configured as required for your use i.e add regions. avatars, users etc_  

- A single 256x256 region that should be compatible with all viewers. This is the default 'landing area' region 'WoaS'
- A variable 512x512 region that will require a viewer compatible with WhiteCore or OpenSim. (Verified with Firestorm Opensim version)  This region is named 'WoaS512' on the world map. You will need to teleport to this region from the landing area.
 
### Installation  
The packages for Windows and Mac OSX are installer packages.  
- Run your downloaded installer and you will  be guided through the installation process  
- These versions of WhiteCore will run in a terminal (command) window for administration purposes (if required)   

#### Windows _(~64Mb)_
Run the included WoasLauncher application and select the 'WhiteCore Sim' entry
- You may be asked to allow some system and network access. This is safe and normal for operation
#### Mac OSX _(~74Mb)_
Run/launch the WhiteCore application from the Applications folder, LaunchPad, Alfred etc.  
- You may be asked to allow system event access for Finder. This is safe and normal for operation
#### Linux _(~64Mb)_
The Linux package requires un-taring in a terminal window after which you will have a 'WhiteCoreSim' directory
- Extract the archive `~$ tar -xzf WoaS_x64_Linux.tgz`
- Change to the WhiteCoreSim directory `~$ cd WhiteCoreSim`
- Startup WhiteCore   `~$ ./sim_console.sh`
#### Linux VM _(~700Mb)_
The WoaS_x64_Linux_VM.ova is a minimal virtual machine based on Alpine with all necessary libraries and WhiteCore installed  
The VM was created in VirtualBox and exported using the Open Virtualization Format 2.0 standard 
- Import the VM to your choice of VM manager and run
- Log in using the user account whitecore, password whitecore  _(A root user of root/root is also available for maintenance)_
- Change to the WhiteCoreSim directory `~$ cd WhiteCoreSim`
- Startup WhiteCore   `~$ ./sim_console.sh`

### Logging In
After WhiteCore has fully started, use your favourite viewer of choice to  'your_local_IP_address:8002'  or 'localhost:8002' / '127.0.0.1:8002'  
You will need to add this grid address in the viewer preferences grid manager prior to login
 - Add your  '<your system IP address>:8002'  to the 'Add a new grid' entry and click 'Apply' to query your sim and retrieve the required details.  Your WhiteCore instance will be identified by 'My WhiteCore Simulator' 

login URI      : http://your_local_IP_address:8002   or   http://127.0.0.1:8002  or  http:localhost:8002
User Name  : Simona Stick   (female avatar)
User Name  : Simon Stick     (male avatar)
Password    : 123                   (for both avatars)

All feedback is welcomed via the discussions pages here, Libera Chat or MeWe _(Details in the readme files)_  
Greythane

<hr /><em>This discussion was created from the release <a href='https://github.com/WhiteCoreSim/WhiteCore-Dev/releases/tag/0.9.6-RC1'>Testing WoaS packages</a>.</em>

**Comment by CharleneMcnally** [comment]
Anyone testing this yet?
